gossip/src/cluster_info.rs

```diff
@@ -1348,7 +1351,7 @@ impl ClusterInfo {
         .filter_map(|(addr, data)| make_gossip_packet(addr, &data, &self.stats))
         .for_each(|pkt| packet_batch.push(pkt));
         if !packet_batch.is_empty() {
-            sender.send(packet_batch)?;
+            _ = sender.try_send(packet_batch);
```

Comment:
why does this not update `gossip_packets_dropped_count`? We also don't return a Result here like we used to on Err. Do other code paths return a result?